superset/assets/stylesheets/superset.less

```diff
@@ -139,6 +139,14 @@ span.title-block {
 }
 div.navbar {
   z-index: 999;
+  .dropdown-menu .fineprint{
+    padding: 10px 20px 0 20px;
+    color: @gray-light;
+    font-size: 1rem;
+    div{
```

Comment:
nit, space between div and `{`

Reply:
Good eyes... I'll open another nit PR that catches a bunch more of these, just for good measure.